oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 sale-workflow
+sale_order_type
```

Comment:
I think I was wrong noting this, that line is unnecessary, the repo dependency was already satisfied. Sorry...
```suggestion
```